src/e2e/kotlin/fi/aalto/cs/apluscourses/e2e/utils/Utils.kt

```diff
@@ -0,0 +1,22 @@
+package fi.aalto.cs.apluscourses.e2e.utils
+
+import com.intellij.remoterobot.RemoteRobot
+import com.intellij.remoterobot.stepsProcessing.StepLogger
+import com.intellij.remoterobot.stepsProcessing.StepWorker
+import java.time.Duration
+
+object StepLoggerInitializer {
```

Comment:
I don't know much about Kotlin, but doesn't it allot top level functions and variables? If so, can we remove the enclosing object?

Reply:
My implementation is actually a crude adaptation of [the example project's logger](https://github.com/JetBrains/intellij-ui-test-robot/blob/master/ui-test-example/src/test/kotlin/org/intellij/examples/simple/plugin/utils/StepsLogger.kt). Their reason to encapsulate it inside an object is probably making sure no one else touches `initialized` field. I think it's fair enough to keep it that way.